src/Microphysics1M.jl

```diff
@@ -124,6 +128,44 @@ function radar_reflectivity(
     return max(FT(-430), log_Z)
 end
 
+"""
+    aspect_ratio_coeffs(snow_shape, mass, area, density)
+
+ - `snow_shape` - a struct specifying assumed snow particle shape (Oblate or Prolate)
+ - `mass` - a struct with assumed m(r) power law relation parameters
+ - `area` - a struct with assumed a(r) power law relation parameters
+ - `ρᵢ` particle density
+
+Returns coefficients of the implied power law relationship between aspect ratio
+and particle diameter ϕ(D) = ϕ₀ D^α
+Also returns the coefficient for the aspect ratio in Chen 2022 terminal velocity
+parameterization (κ=1/3 for oblate and κ=-1/6 for prolate).
+"""
+function aspect_ratio_coeffs(
+    snow_shape::Oblate,
+    (; r0, m0, me, Δm, χm)::CMP.ParticleMass{FT},
+    (; a0, ae, Δa, χa)::CMP.ParticleArea{FT},
+    ρᵢ::FT,
+) where {FT}
+    # ϕ(r) = 3 * sqrt(FT(π)) * mᵢ(r) / (4 * ρᵢ * aᵢ(r)^(3/2)
+    α = me + Δm - 3/2 * (ae * Δa)
```

Comment:
Shouldn't the second term be adding ae and \Delta a?

Reply:
Yes! Thank you